frontend/src/components/activePage/ActiveLanding/ActiveLanding.tsx

```diff
@@ -0,0 +1,94 @@
+import { FC, useState } from 'react';
+import { Col, Switch } from 'antd';
+import WorkspacesTable from '../Workspaces/WorkspacesTable';
+import TemplatesTable from '../Templates/TemplatesTable';
+import Box from '../../common/Box';
+
+export interface IWorkspace {
+  key: string;
+  id: string;
+  name: string;
+  templates: Array<ITemplate>;
+}
+
+export interface ITemplate {
+  key: string;
+  id: string;
+  name: string;
+  workspace: string;
+  nActiveMachines: number;
+}
+
+export const templates: Array<ITemplate> = [
+  {
+    key: 'LANDC-labs',
+    id: 'LANDC-labs',
+    name: 'Laboratorio di Reti Locali e Data Center',
+    workspace: '01SQOOV',
+    nActiveMachines: 3,
+  },
+  {
+    key: 'TSR-labs',
+    id: 'TSR-labs',
+    name: 'Laboratorio di Tecnologie e Servizi di Rete',
+    workspace: '02KPNOV',
+    nActiveMachines: 2,
+  },
+  {
+    key: 'LANDC-project',
+    id: 'LANDC-project',
+    name: 'Progetto Design Rete Aziendale',
+    workspace: '01SQOOV',
+    nActiveMachines: 1,
+  },
+];
+
+export const workspaces: Array<IWorkspace> = [
+  {
+    key: '01SQOOV',
+    id: '01SQOOV',
+    name: 'Reti Locali e Data Center',
+    templates: [templates[0], templates[2]],
+  },
+  {
+    key: '02KPNOV',
+    id: '02KPNOV',
+    name: 'Tecnologie e Servizi di Rete',
+    templates: [templates[1]],
+  },
+];
+
+const ActiveLanding: FC = () => {
+  const [managerView, setManagerView] = useState(false);
+  return (
+    <Box
+      header={{
+        size: 'small',
+        center: (
+          <div className="flex justify-end ml-auto">
+            <Switch
```

Comment:
Could you please look for an icon that explain a bit more the concept of manager/user?
Also I would add here the check to hide the switch if the user is never manager.